Hunting Queries/SecurityEvent/RareProcbyServiceAccount.txt

```diff
@@ -6,15 +6,15 @@
 //
 // DataSource: #SecurityEvent
 //
-// Tactics: #Persistence, #Discovery, #LateralMovement , #Initial Access, #Execution,
+// Tactics: #Persistence, #Discovery, #LateralMovement , #InitialAccess, #Execution,
 //
 let List1 = datatable(AccountName:string)["MSSQLSERVER", "ReportServer", "MSDTSServer100", "IUSR"];         
 // Provide a list of service account/ built-in accounts in an environment.
 let List2 = SecurityEvent                                                                                   
 // Self generating a list of Service account using event Id :4624
 | where TimeGenerated >= ago(1d)
 | where EventID == "4624"
-| where Account !contains "$" and Account !contains "Local SYSTEM" and Account !contains  "Local SERVICE" and Account !contains  "Network SERVICE" and Account !contains "NT AUTHORITY" and Account !contains "NT-AUTORIT�T"
+| where Account !contains "$" and Account !contains "Local SYSTEM" and Account !contains  "Local SERVICE" and Account !contains  "Network SERVICE" and Account !contains "NT AUTHORITY" and Account !contains "NT-AUTORITÄT"
```

Comment:
A better approach to handle localization would be to exclude these accounts using their SID

Reply:
Shain will update the query.